tests/portability/cmsis_rtos_v1/src/kernel_apis.c

```diff
@@ -48,12 +58,12 @@ void test_kernel_systick(void)
 	diff = (uint32_t)k_cyc_to_ns_floor64(stop_time -
 					 start_time) / NSEC_PER_USEC;
 
-	/* Check that it's within 1%.  On some Zephyr platforms
+	/* Check that it's within accepted tolerance. On some Zephyr platforms
 	 * (e.g. nRF5x) the busy wait loop and the system timer are
```

Comment:
The split documentation is not what I had in mind.

Everything in this comment should be up at the point where `TOLERANCE_PPC` is defined: why we need tolerance at all (historically, because Nordic clocks had slew that became visible when clock rates exceeded 100 Hz 33c64c2578d13bd5f63a777077f85a44f966d86e), and why Nordic needs an insanely lax tolerance to pass reliably.

Once `TOLERANCE_PPC` is described it's obvious that we're checking for tolerance and there's no need for a comment here repeating that information.

Reply:
pls check again